tobler/tests/test_area_interpolators.py

```diff
@@ -79,6 +81,32 @@ def test_area_interpolate_categorical():
     assert_almost_equal(area.animal_capybara.sum(), 20, decimal=0)
 
 
+def test_area_interpolate_categorical_dask():
+    sac1, sac2 = datasets()
+    sac1['animal'] = sac1['animal'].astype('category')
+    dsac1 = (
+            dask_geopandas.from_geopandas(sac1, npartitions=2)
+            .spatial_shuffle(by='hilbert', shuffle='tasks')
+    )
+    dsac2 = (
+            dask_geopandas.from_geopandas(sac2, npartitions=2)
+            .spatial_shuffle(by='hilbert', shuffle='tasks')
+    )
+    area = area_interpolate_dask.area_interpolate_dask(
```

Comment:
ah. The issue with the tests is that this is a function not a method. do `area_interpolate_dask()`, not `area_interpolate_dask.area_interpolate_dask()`

Reply:
Thanks! I've changed and I'll how the CI takes it!